client/src/snippets.json

```
@@ -14,8 +25,11 @@
             "    <outputs>",
             "    </outputs>",
             "    <help><![CDATA[",
```

Maybe also add the `tests``placeholder to encourage testing :)
```suggestion
            "    </outputs>",
            "    <tests>",
            "    </tests>",
            "    <help><![CDATA[",
```